src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -505,6 +526,50 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
     }
   }
 
+  override def checkGoogleGroupExists(groupEmail: String): Option[String] = {
+    val directoryService = getDirectoryManager(getDelegatedCredentialForAdminUser)
+    // check if group exists
+    val checkGroupRequest = directoryService.groups.get(groupEmail)
+    val existingGroup = Try(executeGoogleRequest(checkGroupRequest)) match {
+      case Failure(f) =>
+        logger.warn(s"Google Group $groupEmail does not exist")
+        "" // return nothing in case of failure
+      case Success(_) => {
+        groupEmail // return groupEmail
+      }
+    }
+    Option(existingGroup)
+  }
+
+  override def createGoogleGroup(groupEmail: String, targetUserEmail: String): Option[String] = {
+    val newGroup = new Group().setEmail(groupEmail)
+    val directoryService = getDirectoryManager(getDelegatedCredentialForAdminUser)
+
+    val insertRequest = directoryService.groups.insert(newGroup)
+    val setEmail = Try(executeGoogleRequest[Group](insertRequest)) match {
+      case Failure(f) => {
+        logger.warn(s"Could not create new group called $groupEmail")
+        "" // return empty string
+      }
+      case Success(newGroupInfo) => {
+        // add targetUserEmail as member of google group - modeled after `override def addMemberToGroup` in workbench-libs HttpGoogleDirectoryDAO.scala
+        val member = new Member().setEmail(targetUserEmail).setRole("MEMBER").setDeliverySettings("ALL_MAIL")
```

Comment:
`"MEMBER"` and `"ALL_MAIL"` should be vals somewhere, similar to how `HttpGoogleDirectoryDAO` treats it.